conf/airframes/examples/bebop.xml

```diff
@@ -10,6 +10,17 @@
       <module name="udp"/>
     </target>
 
+    <define name="MT9F002_OUTPUT_HEIGHT" value="640" />
```

Comment:
Good point, it isn't that straight forward or obvious. @wilcoschoneveld is it possible for you to make a couple of useful combinations that help a new user? I guess this is also useful for the course.